misk-api/src/main/kotlin/misk/scope/Scope.kt

```diff
@@ -0,0 +1,9 @@
+package misk.scope
+
+interface Scope : AutoCloseable {
```

Comment:
Need to add some comments as this will be interface which should be properly documented